### PR TITLE
Ayush Kalani - code challenge

### PR DESCRIPTION
**Pull request by ayushkalani**

### CODE REVIEW

File: app/controllers/companies_controller.rb [hunk 1]
```diff
@@ -32,6 +32,15 @@ def update
     end
   end  
 
+  def destroy
+    if @company.destroy
+      redirect_to companies_path, notice: t("compaines.destroy", name: @company.name)
+    else
+      Rails.logger.error "Error while destroying company with id: #{@company.id}, errors: #{@company.errors.full_messages}"
```
Comment (ortegacmanuel) [inline app/controllers/companies_controller.rb:39]: @ayushkalani A nice to have these logs! :+1:

File: app/views/companies/_form.html.erb [hunk 2]
```diff
@@ -1,6 +1,14 @@
 <div class="py-5">
   <%= form_for @company do |f| %>
 
+    <% if @company.errors.any? %>
```
Comment (ortegacmanuel) [inline app/views/companies/_form.html.erb:4]: @ayushkalani Good work rendering error messages here! :rocket:

File: config/initializers/load_zip_codes.rb [hunk 3]
```diff
@@ -0,0 +1 @@
+ZipCodes.load unless Rails.env.development?
```
Comment (ortegacmanuel) [inline config/initializers/load_zip_codes.rb:1]: :+1:

**Comment by ortegacmanuel** [comment]
@ayushkalani Thanks for the submission, we will follow up over email.

File: app/views/companies/_form.html.erb (hunk 2 above)
Comment (ayushkalani) [reply app/views/companies/_form.html.erb:4]: @ortegacmanuel Thanks. The cornerstone of all debugging and troubleshooting.

File: config/initializers/load_zip_codes.rb (hunk 3 above)
Comment (ayushkalani) [reply config/initializers/load_zip_codes.rb:1]: @ortegacmanuel better to have the codes in memory if they are used a lot.

File: app/controllers/companies_controller.rb (hunk 1 above)
Comment (ayushkalani) [reply app/controllers/companies_controller.rb:39]: @ortegacmanuel Thanks. The cornerstone of all debugging and troubleshooting.

**Pull request closed without merging**